assets/js/src/form_editor/store/selectors.jsx

```diff
@@ -14,6 +14,15 @@ export default {
   getFormExports(state) {
     return state.formExports;
   },
+  getFormSettings(state) {
+    return state.formData.settings;
+  },
+  getSegments(state) {
+    return state.segments;
+  },
+  getPages(state) {
```

Comment:
Could we rename those two properties? I was confused about what do they mean. I initially thought that it something that a user selected. What about the name `getAllAvailablePages` and `getAllAvailableSegments`?